src/components/autocomplete/js/autocompleteController.js

```diff
@@ -208,6 +208,17 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   }
 
   /**
+   * Find the target for the wrap element which determines menu width
+   * @returns {*}
+   */
+  function getWrapTarget() {
+    for (var element = $element; element.length; element = element.parent()) {
```

Comment:
I suspect it is for similar reasons `md-autocomplete-snap` is found by walking up the DOM.